lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -360,6 +360,13 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
               - a.third.event
           DESCRIPTION
         },
+        :'ai_monitoring.enabled' => {
+          :default => true,
+          :public => true,
+          :type => Boolean,
+          :allowed_from_server => false,
+          :description => 'If `false`, all LLM (OpenAI, Bedrock) instrumentation will be disabled and no metrics, events, or spans will be sent. AI Monitoring is automatically disabled if `high_security` mode is enabled.'
```

Comment:
What do you think about waiting to include `Bedrock` in this description once we have instrumentation for it? 

Maybe we add a TODO or something near the config to help us remember...

Reply:
That's a good call out! I think I'll make a note for tanna to add it here